net/portmapper/portmapper.go

```diff
@@ -433,127 +432,6 @@ func parsePMPResponse(pkt []byte) (res pmpResponse, ok bool) {
 	return res, true
 }
 
-type ProbeResult struct {
```

Comment:
let's not do this code movement. It's already in a portmapper file in a portmapper package. It's okay if it's a little long. 

If we do want to move things around, it should be in its own commit that doesn't add new code.

Reply:
I've moved the probe result back to the same place, but I deleted the whole probe function so I hope that's fine.